noseparallel/plugin.py

```diff
@@ -11,6 +12,8 @@ def configure(self, options, config):
         super(ParallelPlugin, self).configure(options, config)
         self.total_nodes = int(os.environ.get('CIRCLE_NODE_TOTAL') or os.environ.get('NODE_TOTAL', 1))
         self.node_index = int(os.environ.get('CIRCLE_NODE_INDEX') or os.environ.get('NODE_INDEX', 0))
+        for i in xrange(self.total_nodes):
+            os.environ['NODE_LOAD_%d' % i] = str(0)
```

Comment:
Why is this being stored in an environment variable instead of just a global variable?

I don't think any of this state is actually shared across CircleCI containers: I think that after CircleCI initializes the containers, each one has its own state, since each one is actually a separate virtual machine. I'm hoping that this works because it is *deterministic*, so each virtual machine runs this algorithm itself and generates the same results.

We should probably sanity check that this runs the same number of tests with this approach, versus the other.